config/crd/bases/operators.operatorframework.io_operators.yaml

```diff
@@ -35,10 +35,12 @@ spec:
           spec:
             description: OperatorSpec defines the desired state of Operator
             properties:
-              foo:
-                description: Foo is an example field of Operator. Edit operator_types.go
-                  to remove/update
+              packageName:
+                maxLength: 48
+                pattern: ^[a-z0-9]+(-[a-z0-9]+)*$
```

Comment:
Rules out `camelCase`, but I did see the discussion that this matches all current catalogs.

Reply:
True, but I think there's a good chance we'll be generating cluster resources with these names, and if that's the case we'd need to restrict camel case anyway